feedback/client/src/reviews_components/ratings.jsx

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import faStar from "@fortawesome/fontawesome-free-solid/faStar";
+import StarRatings from "react-star-ratings";
+
+export class Ratings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: props.reviews,
+      // stars: null,
+      avgRating: null,
+      five: {
+        width: null
+      },
+      four: {
+        width: null
+      },
+      three: {
+        width: null
+      },
+      two: {
+        width: null
+      },
+      one: {
+        width: null
+      }
+    };
+  }
+
+  componentWillMount() {
+    // var stars = [];
+    // for (var i = 0; i < 5; i++) {
+    //   stars.push(<FontAwesomeIcon icon={faStar} className="star" />);
+    // }
+    var totalSum = 0;
+    var totalNumRatings = 0;
+    var five = 0;
```

Comment:
could replace all of these individual variable declarations with a single array called bars, and then replace all of the if statements with a single line like 
`bars[Math.floor(rating)] = bars[Math.floor(rating)] + 1 || 1;`